tests/profiling/test_recorder.py

```diff
@@ -2,6 +2,24 @@
 from ddtrace.profiling import event
 from ddtrace.profiling import recorder
 
```

Comment:
I think it'd be worthwhile having tests that verify these limits

Reply:
Added